plugins/player-activity-tracker.js

```diff
@@ -169,7 +169,7 @@ window.plugin.playerTracker.processNewData = function(data) {
     });
 
     // skip unusable events
-    if(!plrname || !lat || !lng || !id || skipThisMessage) return true;
+    if (!plrname || !lat || !lng || !id || skipThisMessage || window.teamStringToId(json[2].plext.team) === window.TEAM_NONE) return true;
```

Comment:
since there are only marker for blue/green, it sounds good enough to filter on TEAM_RES and TEAM_ENL